style.css

```diff
@@ -78,8 +83,187 @@ body {
   color: #505f79;
 }
 
+li {
+  list-style-type: none;
+}
+
 .social-icons li {
   list-style-type: none;
   padding: 0;
   margin: 0;
 }
+
+/* works section */
+
+.works {
+  display: grid;
+  gap: 25vw;
+  padding: 0 1vw;
+  margin-top: 25vw;
+  margin-bottom: 36vw;
+}
+
+.card {
+  width: 90vw;
+  margin: auto;
+  background-color: white;
+  padding: 2vw;
+  border: 1px solid #dfe1e6;
+  border-radius: 1rem;
+  -webkit-border-radius: 1rem;
+  -moz-border-radius: 1rem;
+  -ms-border-radius: 1rem;
+  -o-border-radius: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 4vw;
+  padding-bottom: 5vw;
+}
+
+.card-image img {
+  width: 100%;
+}
+
+.card-title {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12vw;
+  line-height: 44px;
+}
+
+.card-year {
+  display: flex;
+  gap: 3vw;
+}
+
+.card-year li {
+  color: #7a869a;
+  align-self: baseline;
+}
+
+.card .canopy {
+  color: #344563;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 4vw;
+  line-height: 16px;
+  text-transform: uppercase;
+}
+
+.empty-dot {
+  width: 2vw;
+  height: 2vw;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+  -webkit-border-radius: 50%;
+  -moz-border-radius: 50%;
+  -ms-border-radius: 50%;
+  -o-border-radius: 50%;
+}
+
+.card-details {
+  color: #344563;
+}
+
+.card-tech-category {
+  display: flex;
+  gap: 3vw;
+}
+
+.card-tech-category li {
+  color: #6070ff;
+  background: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  -ms-border-radius: 8px;
+  -o-border-radius: 8px;
+}
+
+.see-projects-btn {
+  color: #396df2;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 5vw;
+  line-height: 24px;
+  padding: 8px;
+  width: 40vw;
+  background: transparent;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  -ms-border-radius: 8px;
+  -o-border-radius: 8px;
+}
+
+.see-projects-btn:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.disabled {
+  color: #5e6c84;
+  border: 1px solid #5e6c84;
+}
+
+.see-projects-btn:active {
+  color: white;
+  background-color: #2230d2;
+}
+
+.about-myself {
+  padding: 40.25vw 6vw 78.5vw 6vw;
+  background-color: white;
+  width: 100%;
+  margin: 0;
+  border-radius: 0 25vw 0 0;
+  -webkit-border-radius: 0 25vw 0 0;
+  -moz-border-radius: 0 25vw 0 0;
+  -ms-border-radius: 0 25vw 0 0;
+  -o-border-radius: 0 25vw 0 0;
+}
```

Comment:
- Kindly Make Sure Stick to the design Margins as much as possible to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design, according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma). 

**Original Design**
![Screenshot from 2023-04-28 12-20-35](https://user-images.githubusercontent.com/97834160/235124124-cfc94a34-f1c1-41af-b673-0b54717dc0b9.png)


**Your Design**
![Screenshot from 2023-04-28 12-19-58](https://user-images.githubusercontent.com/97834160/235124442-82b2a4c9-2163-488c-ad89-34e5da398803.png)